xrpl/models/base_model.py

```diff
@@ -15,6 +15,9 @@
 from xrpl.models.required import REQUIRED
 from xrpl.models.types import XRPL_VALUE_TYPE
 
+_DETECT_PASCAL_OR_CAMEL_CASE: Final[Pattern[str]] = re.compile(
```

Comment:
nit:
```suggestion
_PASCAL_OR_CAMEL_CASE: Final[Pattern[str]] = re.compile(
```

Reply:
fixed in [244369c](https://github.com/XRPLF/xrpl-py/pull/694/commits/244369cdc80a959c28f70e23ce7d9ba7df07c890)